src/features/defi/providers/thorchain-savers/components/ThorchainSaversManager/Withdraw/components/Withdraw.tsx

```diff
@@ -146,21 +148,30 @@ export const Withdraw: React.FC<WithdrawProps> = ({ accountId, onNext }) => {
     [amountAvailableCryptoPrecision, assetMarketData.price],
   )
 
-  const getOutboundFeeCryptoBaseUnit = useCallback(async () => {
-    const daemonUrl = getConfig().REACT_APP_THORCHAIN_NODE_URL
-    const maybeInboundAddressData = await getInboundAddressDataForChain(daemonUrl, assetId)
-
-    return maybeInboundAddressData.match({
-      ok: ({ outbound_fee }) => {
-        const outboundFeeCryptoBaseUnit = toBaseUnit(
-          fromThorBaseUnit(outbound_fee),
-          asset.precision,
-        )
-        return outboundFeeCryptoBaseUnit
-      },
-      err: () => undefined,
-    })
-  }, [asset.precision, assetId])
+  const getOutboundFeeCryptoBaseUnit = useCallback(async (): Promise<Result<
+    string,
+    string
+  > | null> => {
+    if (!accountId) return null
+
+    try {
+      // Attempt getting a quote with 100000 bps, i.e 100% withdraw
+      // - If this succeeds, this allows us to know the oubtound fee, which is always the same regarding of the withdraw bps
+      // and will allow us to gracefully handle amounts that are lower than the outbound fee
+      // - If this fails, we know that the withdraw amount is too low anyway, regarding of how many bps are withdrawn
+      const quote = await getThorchainSaversWithdrawQuote({ asset, accountId, bps: '10000' })
+
+      const outboundFee = bnOrZero(
+        toBaseUnit(fromThorBaseUnit(quote.fees.outbound), asset.precision),
+      )
+      const safeOutboundFee = bn(outboundFee).times(105).div(100).toFixed(0)
+      // Add 5% as as a safety factor since the dust threshold fee is not necessarily going to cut it
```

Comment:
```suggestion

      // Add 5% as as a safety factor since the dust threshold fee is not necessarily going to cut it
      const safeOutboundFee = bn(outboundFee).times(105).div(100).toFixed(0)

```

Reply:
We can probably calculate this more accurately with the following formula:
```
const available = expected_amount_out + fees.total (
const minWithdrawBps = fees.outbound / available (round up to 4 decimal places)
const minWithdrawalAmount = available * minWithdrawalBps (round ceil)
``
though probably not worth the effort. maybe update the comment to imply the added 5% is just to ensure a net positive withdrawal after fees (dust amount is irrelevant for this amount)